docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,35 +1,43 @@
-* Replaced the REST client with Go for triggering backups (https://github.com/hazelcast/hazelcast-platform-operator/pull/381[#381])
-* Changed the default repository to Hazelcast Enterprise when the `licenseKeySecret` is present (https://github.com/hazelcast/hazelcast-platform-operator/pull/402[#402])
-* Enabled TLS for the backup server sidecar (https://github.com/hazelcast/hazelcast-platform-operator/pull/350[#350])
-* Added a retry for backup service calls (https://github.com/hazelcast/hazelcast-platform-operator/pull/434[#434])
-* Added `WanReplication` to the API reference (https://github.com/hazelcast/hazelcast-platform-operator/pull/355[#355])
-* Refactored the Multimap and Topic CR data structures and Hazelcast's `configMap` creation (https://github.com/hazelcast/hazelcast-platform-operator/pull/393[#393])
-* Upgraded Hazelcast and Management Center versions to 5.1.4
+* Added connectivity check to WAN Replication CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/541[#541])
+** When creating a WAN Replication CR, a user will be sure that the target cluster is reachable
+* Persist Wan Replication configuration into generated Hazelcast ConfigMap (https://github.com/hazelcast/hazelcast-platform-operator/pull/513[#513])
+** Wan Replication configurations will be stored in Hazelcast ConfigMap. When member containers restart, they will be able to continue the replication
+* Client recreation logic for times the client cannot access the cluster (https://github.com/hazelcast/hazelcast-platform-operator/pull/500[#500])
+** The Platform Operator will restart the Hazelcast client when the cluster is inaccessible
+* Converted existing CR validation to validating webhook (https://github.com/hazelcast/hazelcast-platform-operator/pull/492[#492])
+** This change allows the Operator to report issues with CRs before they are applied
+* Unified the approach for exposing API fields (https://github.com/hazelcast/hazelcast-platform-operator/pull/498[#498])
+** Exposed API fields are now more consistent with each other
+* Removed `h.Spec.Persistence.DataLoadTimeoutSec` parameter (https://github.com/hazelcast/hazelcast-platform-operator/pull/512[#512])
+* Split `restore` and `restore_local` commands (https://github.com/hazelcast/hazelcast-platform-operator/pull/471[#471])
+* Used Go client for cluster activation logic (https://github.com/hazelcast/hazelcast-platform-operator/pull/466[#466])
+* Added tilt-debug support  (https://github.com/hazelcast/hazelcast-platform-operator/pull/528[#528])
+* Added `ClusterIDs` to Phone Home data (https://github.com/hazelcast/hazelcast-platform-operator/pull/527[#527])
+* Changed capabilities at CSV from Seamless Upgrades to Full Lifecycle, which is become visible on OCP OperatorHub dashboard(https://github.com/hazelcast/hazelcast-platform-operator/pull/476[#476])
```

Comment:
```suggestion
* Updated capabilities of Cluster Service Version (CSV) from Seamless Upgrades to Full Lifecycle. Both options now appear on the OCP OperatorHub dashboard (https://github.com/hazelcast/hazelcast-platform-operator/pull/476[#476])
```

Reply:
@angelasimms In this case, both options do not appear on the dashboard. One option just replaced another one.

Reply:
@cheels. I've slightly reworded. Does that work?